test/lib/clients/repositories_test.rb

```diff
@@ -105,4 +105,46 @@
       end
     end
   end
+
+  describe "#SortReposByPriority" do
+    it "returns nil if param is nil" do
+      expect(client.SortReposByPriority(nil)).to eq nil
```

Comment:
NP: I'd slightly prefer `be_nil` instead of `eq nil` as it sounds more like spoken English, but it depends on your taste...

Reply:
well, I do not like be_nil as it makes test less explicit. I want to ensure that it really is nil and not some fancy object, that return true for `nil?`

Reply:
Redefining `#nil?` to return `true` would be one of the dirtiest hacks ever. The code which does that is doomed.